Packages/com.unity.inputsystem/InputSystem/Editor/ProjectWideActions/ProjectWideActionsAsset.cs

```diff
@@ -59,6 +60,8 @@ internal static InputActionAsset CreateNewActionAsset()
             asset.LoadFromJson(json);
             asset.name = kAssetName;
 
+            GetDefaultUIActionMap();
```

Comment:
I would suggest passing in the `json` to this function rather than do another file read inside this function.

Reply:
Also, `CreateNewActionAsset()` only gets called once when opening a project for the first time, so the cached value would mostly not exists for existing projects.